open_humans/tests.py

```diff
@@ -169,39 +173,39 @@ def test_lowercase_unique(self):
                           'USER2', 'other+user2@test.com', 'user2')
 
 
-#class OpenHumansBrowserTests(BrowserTestCase):
-#    """
-#    Browser tests of general Open Humans functionality.
-#    """
-#
-#     def test_create_user(self):
-#         driver = self.driver
-#
-#        driver.get(self.live_server_url)
-#
-#        driver.find_element_by_link_text('Become a member').click()
-#
-#        driver.find_element_by_id('signup-username').clear()
-#        driver.find_element_by_id('signup-username').send_keys('test_123')
-#
-#        driver.find_element_by_id('signup-name').clear()
-#        driver.find_element_by_id('signup-name').send_keys('Test Testerson')
-#
-#        driver.find_element_by_id('email-address').clear()
-#        driver.find_element_by_id('email-address').send_keys(
-#            'test@example.com')
-#
-#        driver.find_element_by_id('signup-password').clear()
-#        driver.find_element_by_id('signup-password').send_keys('testing123')
-#
-#        driver.find_element_by_id('signup-password-confirm').clear()
-#        driver.find_element_by_id('signup-password-confirm').send_keys(
-#            'testing123')
-#
-#        driver.find_element_by_name('terms').click()
-#
-#        driver.find_element_by_id('create-account').click()
-#
-#        self.assertEqual(
-#            'Please verify your email address',
-#            driver.find_element_by_css_selector('h3.panel-title').text)
+# class OpenHumansBrowserTests(BrowserTestCase):
```

Comment:
Are we holding on to this because it might be usable again at some point? Maybe add a comment on why the commented code is retained but commented out?